gamemode/core/libs/sh_faction.lua

```diff
@@ -80,6 +80,31 @@ function nut.faction.getIndex(uniqueID)
 	return nut.faction.teams[uniqueID] and nut.faction.teams[uniqueID].index
 end
 
+function createNutJob(index, name, color, default, models)
```

Comment:
This function shouldn't be included directly in _G but instead in the `nut.faction` namespace.